recipes/gsw-xarray/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "gsw-xarray" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gsw-xarray-{{ version }}.tar.gz
+  sha256: 867cd4b3bbcd18cfcb11caf7778ce6beb98f26e7a6ec459740bde8dea85b6f56
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --use-deprecated=out-of-tree-build
  number: 0

requirements:
  host:
    - pip
    - poetry-core
    - python >=3.8
```

https://github.com/conda-forge/conda-forge.github.io/pull/1540

---

Thanks @carterbox  for the help
Is the --use-deprecated=out-of-tree-build necessary? In the pip doc it seems that it will be removed in pip version 22.1 (https://pip.pypa.io/en/latest/topics/local-project-installs/#build-artifacts)